pg/pg_test.go

```diff
@@ -31,9 +25,40 @@ func TestNewPG(t *testing.T) {
 	err = db.Conn.QueryRow("SELECT 1 + $1", 1).Scan(&response)
 
 	if response != 2 || err != nil {
-		cleanup()
+		os.Remove("pg_test.env")
 		t.Fatalf("Error preforming simple query:\n%s", err)
 	}
 
-	cleanup()
+	pg = db
+
+	os.Remove("pg_test.env")
+}
+
+func TestLoadQueryMap(t *testing.T) {
+	os.Mkdir("queries_test", 0755)
+	os.WriteFile("queries_test/one.sql", []byte("SELECT 1 + $1;"), 0755)
+	os.Mkdir("queries_test/nested", 0755)
+	os.WriteFile("queries_test/nested/two.sql", []byte("SELECT $1 + $2;"), 0755)
+
+	pg.LoadQueryMap("queries_test")
+
+	one, ok := pg.QueryMap["queries_test/one"]
+	if ok {
+		if one != "SELECT 1 + $1;" {
```

Comment:
If we are using writefile for these instead of having a test_data dir, we might as well take advantage of that by capturing the values as string constants instead of copypasting.